jsapp/js/components/transferProjects.module.scss

```diff
@@ -0,0 +1,65 @@
+@use 'scss/sizes';
+@use '~kobo-common/src/styles/colors';
+
+.root {
+  background: colors.$kobo-white;
+  padding: sizes.$x20;
+  margin: sizes.$x24 sizes.$x40;
+  border: 1px solid colors.$kobo-gray-92;
+  border-radius: sizes.$x5;
+}
+
+.bar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  button {
+    max-width: sizes.$x180;
+    height: sizes.$x38;
+  }
+}
+
+.description {
+  margin-right: sizes.$x16;
+}
+
+.copy {
+  margin-top: sizes.$x8;
+}
+
+.modalBody {
+  line-height: sizes.$x24;
```

Comment:
Why do we override `line-height` on everything inside modal? Shouldn't this be applied only to `<p>`? Should we expect all paragraphs (all `<p>` functioning as actual paragraphs, not all `<p>` in general) in UI to have this line height? Good to ask Tessa probably - but regardless, I think we need to apply these styles more precisely.